src/main/java/org/broadinstitute/hellbender/tools/exome/plotting/PlotSegmentedCopyRatio.java

```diff
@@ -66,7 +87,8 @@
             doc = "File containing the reference sequence dictionary (used to determine relative contig lengths). " +
                     "Contigs will be plotted in the order given. " +
                     "Contig names should not include \"" + CONTIG_DELIMITER + "\"." +
-                    "Data on contigs not in the dictionary will not be plotted and a warning will be thrown.",
+                    " Only data for contigs represented by the dictionary will be plotted. " +
```

Comment:
Just a minor nitpick, but I would add the missing space to the end of the previous line (rather than the beginning of this line) for consistency.